packages/fetchai/skills/erc1155_deploy/tests/test_handlers.py

```diff
@@ -384,7 +381,7 @@ def test_handle_transaction_digest(self):
             LedgerApiDialogue,
             self.prepare_skill_dialogue(
                 dialogues=self.ledger_api_dialogues,
-                messages=self.list_of_ledger_api_messages[:3],
+                messages=self.list_of_ledger_api_messages[2:3],
                 counterparty=LEDGER_API_ADDRESS,
             ),
         )
```

Comment:
slice(2:3) -> [2]
slice(4:5) -> [4]

don't we also need to test the rest of the (now atomic request response) message sequence?

Reply:
@Karrenbelt 
goal of the test is to check skill, not protocol iteself.
`self.prepare_skill_dialogue` -> make a dialogue with a certain state. after the protocol change no need in intermediate states. so state prepration now is only one message instead of three